### DISCUSSION TITLE
39

### DISCUSSION THREAD

**Discussion by giscus** [car-ffeine/car-ffeine.github.io#20] [Announcements]
# 39

카페인 서비스를 개발하면서 가장 많이 받은 피드백 중 하나는 사용자 경험이 반드시 필요하다는 것이었습니다.

https://car-ffeine.github.io/39

**Comment by green-kong** [comment]
기가 막히네요!
서비스에 대한 진심이 느껴집니다.
멋있어요 카페인!

**Reply by gabrielyoon7** [reply]
헉.. 아닙니다...!!
블로그 방문해주셔서 감사합니다!!

**Reply by drunkenhw** [reply]
다음에는 전기차타고 성수 카페 가죠

**Reply by gabrielyoon7** [reply]
성수는 주차할 곳이 있을까요 . . . ?? ㅠㅠ

**Comment by Gilpop8663** [comment]
서비스를 만들고 사용자 경험을 개선하기 위해서 차를 빌려본 점이 너무 멋있어요

**Reply by gabrielyoon7** [reply]
ㅋㅋㅋㅋ 감사합니다

**Comment by jinny-park** [comment]
Good~

**Reply by gabrielyoon7** [reply]
감사합니다
